packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts

```diff
@@ -515,3 +528,25 @@ export const WithRightAlignment = {
     placeholder: "Right alignment...",
   },
 } satisfies Story;
+
+/**
+ * This example shows a single select with options containing additional data, passed as slot.
+ */
+export const WithAdditionalData = {
```

Comment:
We already have an example with custom data, would prefer to replace it with your example